src/main/webapp/resources/js/components/samples/components/fastqc/FastQC.jsx

```diff
@@ -0,0 +1,95 @@
+import { Button, Modal, Space, Typography } from "antd";
+import React from "react";
+import { FastQCMenu } from "./FastQCMenu";
+
+const { Text } = Typography;
+import { IconArrowLeft } from "../../../icons/Icons";
+import { clearFastQCData, setFastQCDetails } from "./fastQCSlice";
+import { useDispatch, useSelector } from "react-redux";
+import { getFastQCDetails } from "../../../../apis/files/sequence-files";
+import { InfoAlert } from "../../../alerts";
+
+/**
+ * Function to render fastqc modal
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export function FastQC() {
+  const dispatch = useDispatch();
+
+  const qcEntryTranslations = {
```

Comment:
Constants like this could actually just be placed at the top of file or in a different `contstans` file, reason being is that this object will get set up for each instance, which is not necessary and just takes up memory.

Reply:
Moved to top of file in 22be3ade657236165bdb644ecbb598e66d6e8328. Doesn't need to be in a constants file as it's the only place it's being used